pkg/operator/controllers/genevalogging/const.go

```diff
@@ -73,7 +73,7 @@ const (
 [FILTER]
 	Name grep
 	Match containers
-	Regex NAMESPACE ^(?:default|kube-.*|openshift|openshift-.*)$
+	Regex NAMESPACE ^(?:default|kube-.*|openshift|(?!openshift-(logging|gitops))(openshift-.*))$
```

Comment:
Should we exclude `openshift-user-workload-monitoring` as well?

https://docs.openshift.com/container-platform/4.10/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects

Reply:
Yeah, this is a good idea.

Reply:
Thanks @cadenmarchese for pointing this out. Since the linked work item is to exclude any logs from customer workloads, it seems fitting to exclude the said namespace as well.